apps/zbugs/server/notify.ts

```diff
@@ -71,35 +64,44 @@ export async function notify(
   const modifierUser = await tx.query.user.where('id', modifierUserID).one();
   assert(modifierUser);
 
-  const recipients = await gatherRecipients(
-    // TODO: we shouldn't have to assert and cast this given this is called from `server-mutators`.
-    tx as ServerTransaction<Schema, postgres.TransactionSql>,
-    issueID,
-    isAssigneeChange,
-    previousAssigneeID,
-  );
+  const recipients = await gatherRecipients(tx, issueID, modifierUserID);
```

Comment:
Maybe call this `emailRecipients` to make more clear since it's not used for discord.